packages/block-editor/src/components/block-navigation/index.js

```diff
@@ -67,10 +67,11 @@ export default compose(
 		} = select( 'core/block-editor' );
 		const selectedBlockClientId = getSelectedBlockClientId();
 		return {
-			rootBlocks: getBlocks(),
+			rootBlocks: getBlocks( '', { includeControlledInnerBlocks: true } ),
```

Comment:
Calling selectors with inline objects is never a good idea in terms of performance, not sure it's the source of the issue but it's definitely something to avoid in general (breaks memoization)